docs/conf.py

```diff
@@ -152,7 +152,7 @@
         u"MetaDeploy Documentation",
         author,
         "cookiecutterproject_name",
-        """A short description of the project.""",
+        """Web-based tool for installing Salesforce products""",
```

Comment:
No reason for this to be in tripquotes; it's not long and not even expected to be long.